translate/cloud-client/translate_v3_translate_text_with_glossary.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_translate_text_with_glossary]
+
+from google.cloud import translate_v3
+
+
+def translate_text_with_glossary(
+    text, source_language, target_language, project_id, glossary_id
+):
+    """Translates a given text using a glossary."""
+
+    client = translate_v3.TranslationServiceClient()
+
+    # text = 'Hello, world!'
+    # source_language = 'en'
+    # target_language = 'fr'
+    # project_id = '[Google Cloud Project ID]'
+    # glossary_id = '[YOUR_GLOSSARY_ID]'
+    contents = [text]
+    parent = client.location_path(project_id, "us-central1")
+
+    glossary = client.glossary_path(
+        project_id, "us-central1", glossary_id  # The location of the glossary
+    )
+
+    glossary_config = translate_v3.types.TranslateTextGlossaryConfig(glossary=glossary)
```

Comment:
nit: Some comments with additional context on statements like this might be useful. I have no idea what the glossary is or what value it provides from reading this snippet.